scripts/install.gs

```diff
@@ -18,6 +18,16 @@ if *goGet {
 	Printfln("go get -u -v %s/%s", GCSE, a)
 	MustSucc(Bash("go get -u -v %s/%s", GCSE, a))
   }
+
+  additionalPackages := []string{
```

Comment:
Nope, when I tested this morning:
- go test (called later in `install.gs`) complained about missing `github.com/golangplus/testing/assert`
- crawler complained about `fmt` and `heap`